### PR TITLE
updog and mangrove basket

### PR DESCRIPTION
**Pull request by BasiqueEvangelist**

### CODE REVIEW

File: src/main/java/io/wispforest/affinity/object/AffinityBlocks.java [hunk 1]
```diff
@@ -54,6 +55,7 @@ public class AffinityBlocks implements BlockRegistryContainer {
     @Tab(AffinityItemGroup.MAIN) public static final Block SUNSHINE_MONOLITH = new SunshineMonolithBlock();
     @Tab(AffinityItemGroup.MAIN) public static final Block ARCANE_TREETAP = new ArcaneTreetapBlock();
     @Tab(AffinityItemGroup.MAIN) public static final Block ASSEMBLY_AUGMENT = new AssemblyAugmentBlock();
+    @NoBlockItem @Tab(AffinityItemGroup.MAIN) public static final Block MANGROVE_BASKET = new MangroveBasketBlock();
```
Comment (gliscowo) [inline src/main/java/io/wispforest/affinity/object/AffinityBlocks.java:58]: `BlockItemProvider` moment?

File: src/main/java/io/wispforest/affinity/object/AffinityItems.java [hunk 2]
```diff
@@ -49,6 +49,7 @@ public class AffinityItems implements ItemRegistryContainer {
     public static final Item AZALEA_SIGN = new SignItem(settings(AffinityItemGroup.NATURE).maxCount(16), AffinityBlocks.AZALEA_SIGN, AffinityBlocks.AZALEA_WALL_SIGN);
     public static final Item AZALEA_BOAT = new BoatItem(false, AffinityBlocks.AZALEA_BOAT_TYPE, settings(AffinityItemGroup.NATURE).maxCount(1));
     public static final Item AZALEA_CHEST_BOAT = new BoatItem(true, AffinityBlocks.AZALEA_BOAT_TYPE, settings(AffinityItemGroup.NATURE).maxCount(1));
+    public static final Item MANGROVE_BASKET = new MangroveBasketItem();
```
Comment (gliscowo) [inline src/main/java/io/wispforest/affinity/object/AffinityItems.java:52]: This should then be moved into the override in `MangroveBasketBlock`

**Review by gliscowo — CHANGES_REQUESTED** [review]

File: src/main/java/io/wispforest/affinity/object/AffinityBlocks.java (hunk 1 above)
Comment (BasiqueEvangelist) [reply src/main/java/io/wispforest/affinity/object/AffinityBlocks.java:58]: since when is that a thing

Comment (gliscowo) [reply src/main/java/io/wispforest/affinity/object/AffinityBlocks.java:58]: [image](https://github.com/wisp-forest/affinity/commit/e271b62b0d4929624a0f79776a555801d73e7b85)

**Review by gliscowo — APPROVED** [review]

**Pull request merged**